lib/util/template/index.ts

```diff
@@ -143,6 +144,8 @@ export const allowedFields = {
   version: 'The version number of the changelog',
   versioning: 'The versioning scheme in use',
   versions: 'An array of ChangeLogRelease objects in the upgrade',
+  vulnerabilitySeverity:
+    'The vulnerability severity for a vulnerability alert upgrade',
```

Comment:
We should document which levels are available to the user.

You should be able to dynamically template in the levels.

Reply:
Can you elaborate more please @secustor.  

Are you thinking it should mention `LOW, MODERATE, HIGH, CRITICAL` for example?